src/styles/Nav.scss

```diff
@@ -1,7 +1,5 @@
-.main-nav{
-  margin-left: 20px;
-}
-
-a {
-  text-decoration: none;
```

Comment:
I removed this, because I think underlining is a nice indicator that something is a link for accessibility purposes. Open to discussing it!